include/extractor/edge_based_graph_factory.hpp

```diff
@@ -84,7 +84,7 @@ class EdgeBasedGraphFactory
     EdgeBasedGraphFactory &operator=(const EdgeBasedGraphFactory &) = delete;
 
     explicit EdgeBasedGraphFactory(std::shared_ptr<util::NodeBasedDynamicGraph> node_based_graph,
-                                   const CompressedEdgeContainer &compressed_edge_container,
+                                   CompressedEdgeContainer &compressed_edge_container,
```

Comment:
@MoKob I think the EdgeBasedGraphFactory now calls methods that refactor the internal storage of the compressed geometries from the single-direction geometries to the bidirectional geometries. I think this happens in the `InsertEdgeBasedNode` method because at that point we have access to node-based edges that point to the same geometry but in differetn directions.

Is there a better time to create the internal bidirectional geometry tables?